client/src/featureform/register.py

```
@@ -4876,6 +4886,137 @@
+        request = self._create_prune_request(source, variant, resource_type)
+
+        # Send the request to delete the resource
+        self._stub.PruneResource(request)
+        print("Deleting resource async")
+
+    def _create_prune_request(
+        self,
+        source: Union[DeletableResource, str],
+        variant: Optional[str] = None,
+        resource_type: Optional[ResourceType] = None,
+    ) -> metadata_pb2.PruneResourceRequest:
```

Ruff (0.8.2)

`metadata_pb2` may be undefined, or defined from star imports

(F405)